kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableConfig.java

```diff
@@ -254,6 +254,20 @@ public static Map<String, String> validateProperties(Map<String, String> configu
       String key = kv.getKey().toLowerCase(Locale.ROOT);
       String value = kv.getValue();
       if (key.startsWith("delta.") && VALID_PROPERTIES.containsKey(key)) {
+        // Disable `columnMapping` and `icebergCompatV2` for Delta 3.3 release
```

Comment:
Should we do this check after we filter out to only have the "new properties"? I'm not super familiar with the expected semantics with the table properties code on writes, but it seems like there might be existing properties here for a table that already has column mapping enabled?

Reply:
Very good point. Just added a fix and test to make sure it is not writable for existing column mapping enabled tables.